src/async/serializeAsync.ts

```diff
@@ -278,6 +284,8 @@ export function createTsonSSEResponse(opts: TsonAsyncOptions) {
 				"Cache-Control": "no-cache",
 				Connection: "keep-alive",
 				"Content-Type": "text/event-stream",
+				// prevent buffering by nginx
+				"X-Accel-Buffering": "no",
```

Comment:
In our json stream we handle partial strings (we buffer until linebreaks) , what happens to the response chunks if/when nginx buffers? Could we just handle it instead or do they break SSE?

Reply:
Yeah, buffering breaks streams, so you get a blocking situation. I believe chunks should stay intact, i.e. you get on the client what you sent from the server. See the [first paragraph](https://gist.github.com/v-kolesnikov/8e5e4bc72726b8c190ca487ed213365e#the-web-server) under 'the-web-server' for more